### PR TITLE
SC-6278 - validate create user forms

### PR DESCRIPTION
**Pull request by pkleybolte**
# Description

## Links to Tickets or other pull requests
https://ticketsystem.hpi-schul-cloud.org/browse/SC-6278

## Screenshots of UI changes
AddTeacherServerError

AddTeacherValidationError

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)
[](url)

### CODE REVIEW

**Comment by pkleybolte** [comment]
@schul-cloud-bot update screenshots

**Comment by pkleybolte** [comment]
@schul-cloud-bot update screenshots

File: src/components/atoms/ErrorMessage.vue [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	align-items: center;
+	font-size: var(--text-sm);
+	color: var(--color-danger);
+}
+.error-icon {
+	width: 24px;
+	height: 24px;
+	margin-right: var(--space-xs-2);
+}
+</style>
```
Comment (CordlessWool) [inline src/components/atoms/ErrorMessage.vue:33]: shouldn't it more general? So you can also do warnings, inforamtion and success?

**Review by sarahschaefer — CHANGES_REQUESTED** [review]
Hi, As it looks like teh checkbox is aligned to the top of the writing, I would also align the error icon to the top

**Comment by pkleybolte** [comment]
@schul-cloud-bot update screenshots


> Hi, As it looks like teh checkbox is aligned to the top of the writing, I would also align the error icon to the top

No, the checkbox and its label are vertically centered as is the error icon

File: src/components/atoms/ErrorMessage.vue (hunk 1 above)
Comment (pkleybolte) [reply src/components/atoms/ErrorMessage.vue]: Done

**Review by sarahschaefer — CHANGES_REQUESTED** [review]
Then please top-align both, or even better centre-align only to the first line (if possible :-))

There are some rules for the input field error handling (see attachment) 
Screenshot 2020-10-06 at 15 37 33

Thank you!

**Comment by pkleybolte** [comment]
@schul-cloud-bot update screenshots

**Review by Metauriel — APPROVED** [review]

**Review by sarahschaefer — CHANGES_REQUESTED** [review]
The Warning Icon is to far down

**Review by sarahschaefer — APPROVED** [review]

**Review by CeEv — APPROVED** [review]

**Pull request merged**